### PR TITLE
{lib}[foss/2019b] imgaug v0.4.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by edmondac**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@edmondac problems with tests should be fixed with the changes in #11069, please sync your PR branch with `develop`?

You will need to roll back bb3660e though to avoid a merge conflict...

**Comment by edmondac** [comment]
Test report by @bear-rsg
**SUCCESS**
Build succeeded for 0 out of 0 (2 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0efab95cee72480e3dfce66828288f6e for a full test report.

**Comment by edmondac** [comment]
Test report by @bear-rsg
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/edaa6c661442e2ac5d6f91340fe109a0 for a full test report.

File: easybuild/easyconfigs/i/imgaug/imgaug-0.4.0_openvc_requirement.patch [hunk 1]
```diff
@@ -0,0 +1,27 @@
+--- imgaug-0.4.0/setup.py.orig	2020-02-05 20:37:03.000000000 +0000
```
[the remark this answers is not in the document]
Comment (edmondac) [reply easybuild/easyconfigs/i/imgaug/imgaug-0.4.0_openvc_requirement.patch:1]: Done

**Comment by edmondac** [comment]
Test report by @bear-rsg
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ab297a36fe54283f4d18153bdb063b47 for a full test report.

**Comment by edmondac** [comment]
Test report by @bear-rsg
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ffbea0c3997380b3878f41d95317df95 for a full test report.

File: easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Pillow', '6.2.1'),
+    ('matplotlib', '3.1.1', versionsuffix),
+    ('scikit-image', '0.16.2', versionsuffix),
+    ('OpenCV', '4.2.0', versionsuffix),
+    ('Shapely', '1.7.0', versionsuffix),
+    ('imageio', '2.6.1', versionsuffix),
```
Comment (edmondac) [inline easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2019b-Python-3.7.4.eb:21]:
```suggestion
    ('imageio', '2.9.0', versionsuffix),
```

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node111.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/a11b18fa9e9986da6257f0cbd1c50da8 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node368.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/1ef57e23fc411477b55b9df08ce418fe for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @edmondac !

**Pull request merged**